app/wdio-validation/page-objects/showCaseComponentsPage.ts

```diff
@@ -0,0 +1,55 @@
+import { BasePage, InputField, Label, Button } from 'systelab-components-wdio-test';
+
+export class ShowCaseComponentsPage extends BasePage {
+  constructor() {
+    super('showcase-components');
+  }
+
+  public getTextBoxByClassName(): InputField {
+    return new InputField(this.byClassName('form-control'));
+  }
+
+  public getTextBoxByCss(): InputField {
+    return new InputField(this.byCSS('.form-control'));
+  }
+
+  public getTextBoxById(): InputField {
+    return new InputField(this.byId('full-width-input'));
+  }
+
+  public getIconQuestionButton(): Button {
+    return new Button(this.byCSS('.icon-question'));
+  }
+
+  public getTextBoxDisabledById(): Button {
```

Comment:
The name of the method, maybe should be referred to a 'Button'.

Reply:
@JordiCela good catch!
But as the code inside is actually locating 'disabled-input', it appears that the method's name would be convenient, and it would be rather the return type and the constructor that might need to be changed to InputType.

Reply:
Reviewing that part of the code we figure out that this method was duplicated and unused. 
Now it would be fixed.

Thanks a lot! :-)

Reply:
You are welcome ;)